### ISSUE TITLE
New ML Algorithms

### ISSUE THREAD

**Issue by lejacome** [schollz/find#208]
Hi, 
there a way to implements new machine learning algorithms(bagging, k-near neighbor, adaboost) from *.py file or I need to do it directly in golang

**Comment by schollz** [comment]
The easiest thing to do is to edit the Python file. If you are still interested, I've done this a little bit already. I am working on updating FIND to include bluetooth and a whole assortment of ML algs.

**Comment by lejacome** [comment]
Yes sure... I'm working implementing new algorithms and see if it's possible to mix wifi and ble to improve the accuracy system.

**Comment by schollz** [comment]
Haha I also did that too! It's at GitHub.com/de0gee 

I don't have much usable documentation at the moment, but if your interested I can write something to point you in the write direction
